packages/form-js-viewer/src/render/components/form-fields/Textarea.js

```diff
@@ -0,0 +1,89 @@
+import { isArray, isObject } from 'min-dash';
+import { useContext } from 'preact/hooks';
+
+import { FormContext } from '../../context';
+
+import Description from '../Description';
+import Errors from '../Errors';
+import Label from '../Label';
+
+import {
+  formFieldClasses,
+  prefixId
+} from '../Util';
+
+const type = 'textarea';
+
+
+export default function Textarea(props) {
+  const {
+    disabled,
+    errors = [],
+    field,
+    value = ''
+  } = props;
+
+  const {
+    description,
+    id,
+    label,
+    validate = {}
+  } = field;
+
+  const { required } = validate;
+
+  const onInput = ({ target }) => {
+
+    props.onChange({
+      field,
+      value: target.value
+    });
+
+    autoSizeTextarea(target);
```

Comment:
When I first type in something, the text area got's expanded immediately, although it's not needed. Not sure whether it's a [playground demo](https://demo-textarea--camunda-form-playground.netlify.app/) issue though.

![Kapture 2022-11-01 at 13 33 57](https://user-images.githubusercontent.com/9433996/199233870-70101992-eb8e-48f7-be57-232486d18441.gif)

Reply:
Behavior is now very consistent